gem/tests/test_views.py

```diff
@@ -369,6 +378,24 @@ def test_comment_shows_user_display_name(self):
         self.assertContains(response, "this is my alias")
         self.assertNotContains(response, "tester")
 
+    def test_anonymous_comment_translation(self):
+        self.user.profile.alias = 'this is my alias'
```

Comment:
@sewagodimo If you are testing for Anonymous I don't think you need to add an alias to the user. However if you need to do that you have to save the user after assigning the Alias `self.user.profile.save()` but in this case you don't need `self.user.profile.alias = 'this is my alias'`